### ISSUE TITLE
FSE Block Support

### ISSUE THREAD

**Issue by jasontucker** [pfefferle/wordpress-webmention#340]
I'd love to see webmentions have their own block allowing us to specify where we want this block to be located in Full Site Editor on the template page.

**Comment by electricbrick** [comment]
Agreed. The ability to curate the appearance of a webmentions block/component would go a long way in making this palatable for those modern WordPress users seeking to put the focus back on our own sites while maintaining connections with social networks... Right now it's working decently on my site, but Jason quite simply can't get webmentions to materialize in his comments section. We're both using block themes from the same author, as well.

<img width="724" alt="Screenshot 2022-10-28 at 10 01 54 PM" src="https://user-images.githubusercontent.com/2481417/198813483-1f76b966-ecfb-4702-860e-7158d5fb2117.png">

**Comment by pfefferle** [comment]
I totally agree, but I have the often discussed problem, that I am a PHP developer, have no idea how FSE and the block editor works and have only limited skills in JavaScript and no react experience at all. But I would love to support if someone would like to step in to prepare a pull request!
